integration/airflow/tests/integration/tests/airflow/dags/dbt_snowflake.py

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+
+from airflow.operators.bash_operator import BashOperator
+from airflow.utils.dates import days_ago
+
+from openlineage.airflow.utils import JobIdMapping
+
+try:
+    from airflow.utils.db import create_session
+except ImportError:
+    from airflow.utils.session import create_session
+
+# do not need airflow integration
+from airflow import DAG
+from airflow.version import version as AIRFLOW_VERSION
+
+
+def lineage_parent_id(run_id, task):
+    with create_session() as session:
+        job_name = f"{task.dag_id}.{task.task_id}"
+        ids = str(JobIdMapping.get(job_name, run_id, session))
+        return f"{os.getenv('OPENLINEAGE_NAMESPACE')}/{job_name}/{ids}"
+
+
+PROJECT_DIR = "/opt/data/dbt/testproject"
+PROFILE_DIR = "/opt/data/dbt/profiles"
+
+default_args = {
+    'owner': 'datascience',
+    'depends_on_past': False,
+    'start_date': days_ago(7),
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'email': ['datascience@example.com']
+}
+
+if AIRFLOW_VERSION == '2.2.4':
```

Comment:
Hate to be really annoying about this after you *just* changed it, but to make this more reliable in the long term, I think this should really be a `>= 2.2.x` or even `2.x.x`.